test_reduction_pipeline.R

```diff
@@ -0,0 +1,135 @@
+
+taskSurv_toClassif = function(task, cut = NULL) {
```

Comment:
the two functions are not needed now as you did the `PipeOp`s, and the rest from this file can go to `tests` one way or another